content/docs/command-reference/fetch.md

```diff
@@ -17,22 +17,12 @@ positional arguments:
 
 ## Description
 
-Downloads DVC-tracked files from remote storage into the cache of the project
-(without placing them in the <abbr>workspace</abbr>, like `dvc pull` would).
-This makes them available for linking (or copying) into the workspace (refer to
-[`dvc config cache.type`](/doc/command-reference/config#cache)).
+Downloads tracked files and directories from remote storage into the
+<abbr>cache</abbr> (without placing them in the <abbr>workspace</abbr> like
+`dvc pull`). This makes them available for linking (or copying) into the
```

Comment:
if we change this doc, we can now mention that it also makes these objects available to commands like `metrics show`, plots and others that read different commits

I would split `This makes them available for linking (or copying) into the` into a separate paragraph.

Reply:
> mention that it also makes these objects available to commands like metrics show, plots and others

Good idea. I checked `diff` and it doesn't need the cache so it's only `metrics/plots show/diff`. Added a note because once I wrote it it seemed sort of secondary (too specific to metrics and plots).

> I would split This

Since the new text is a note, I didn't split that for now.

Reply:
p.s. I opened iterative/dvc/issues/5039

Reply:
> also makes these objects available to commands like metrics show

Actually, it was already mentioned: "Here are some scenarios... To use comparison commands across different Git commits, for example dvc metrics" so I just added a `dvc plots diff` link there to make the bullet more complete (rolled back the note previously added).